akka-stream/src/main/resources/reference.conf

```diff
@@ -98,12 +98,57 @@ akka {
         fixed-pool-size = 16
       }
     }
+
+    # configure defaults for SourceRef and SinkRef
+    stream-refs {
+      # Default initial demand to be sent along with a SinkRef.
+      #
+      # The receiving side will be prepared to buffer as much as that many elements,
+      # and the sending side can assume to safely send as many elements without receiving
+      # an demand message before.
+      initial-demand = 4
+
+      # Demand is signalled by sending a cumulative demand message ("requesting messages until the n-th sequence number)
+      # Using a cumulative demand model allows us to re-deliver the demand message in case of message loss (which should
+      # be very rare in any case, yet possible -- mostly under connection break-down and re-establishment).
+      #
+      # The semantics of handling and updating the demand however are in-line with what Reactive Streams dictates.
+      #
+      # In normal operation, demand is signalled in response to arriving elements, however if no new elements arrive
+      # within `demand-redelivery-interval` a re-delivery of the demand will be triggered, assuming that it may have gotten lost.
+      demand-redelivery-interval = 500 ms
```

Comment:
perhaps increase to 1s, which is like a typical heartbeat
it will also be sent when there are no stream elements, I guess

Reply:
Sounds good